packages/create-frames/README.md

```diff
@@ -0,0 +1,21 @@
+# Create Frames.js app
+
+Create a new Frames.js app with a single command using predefined templates.
```

Comment:
would be great to document how to test this cli locally

Reply:
It is not really intended to be installed locally to use it. I will document it.